platform-core-module/src/main/java/com/softicar/platform/core/module/file/smb/ISmbDirectory.java

```diff
@@ -15,20 +16,99 @@
 	 */
 	void mkdirs();
```

Comment:
I now you haven't changed this line but I wonder: Shouldn't this method be renamed to `makeDirectories()`?